src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.List;
+import racingcar.exception.GameException;
+
+public class InputView {
+    private final GameException gameException = new GameException();
+
+    private static final String carDelimiter = ",";
+
+    public List<String> inputCars() {
+        try {
+            String input = Console.readLine();
+            List<String> splitCar = List.of(input.split(carDelimiter, -1));
+            gameException.validateCar(splitCar);
+            return splitCar;
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
+            throw new IllegalArgumentException(e);
```

Comment:
```suggestion
            throw new IllegalArgumentException(e.getMessage());
```
예외 메시지 출력을 위와 같이 줄일 수 있습니다!